akka-remote/src/main/scala/akka/remote/artery/compress/DecompressionTable.scala

```diff
@@ -27,6 +31,10 @@ private[artery] final case class DecompressionTable[T](originUid: Long, version:
 
 /** INTERNAL API */
 private[artery] object DecompressionTable {
+
+  val DisabledVersion: Byte = -1
+
   private[this] val _empty = DecompressionTable(0, 0, Array.empty)
   def empty[T] = _empty.asInstanceOf[DecompressionTable[T]]
```

Comment:
Not 100% sure that I got it right, but this is how I understood it. -1 for disabled, 0 is just the empty/starting version, nothing special about it.